src/benchmark.cr

```diff
@@ -117,9 +117,9 @@ module Benchmark
 
   # Returns the time used to execute the given block.
   def measure(label = "") : BM::Tms
-    t0, r0 = Process.times, Time.now
+    t0, r0 = Process.times, Time.monotonic
```

Comment:
Why not use `Time.measure` here?
```crystal
tms0 = Process.times
elapsed = Time.measure do
  yield
end
tms1 = Process.times
BM::Tms.new(tms1.utime - tms0.utime,
  tms1.stime - tms0.stime,
  tms1.cutime - tms0.cutime,
  tms1.cstime - tms0.cstime,
  elapsed.total_seconds,
  label)
```

Reply:
I would hold off on unnecessary changes like that, at least in this PR. They don't simplify anything but do cause errors, as you've pointed out elsewhere.

Reply:
Errors only occur if it involves a return value from the block. Here this doesn't matter.

I think it should be worth advocating the more idiomatic `Time.measure` when possible.

Reply:
This way it's obvious both measurements are related. It also avoids too much refactor for little purpose.